docs/osquery/osquery-placeholder-fields.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[osquery-placeholder-fields]]
+== Use placeholder fields in Osquery queries 
+
+Instead of hard-coding alert and event values into Osquery queries, you can use placeholder fields to dynamically pass this data into queries. Placeholder fields function like replaceable parameters. You can use placeholder fields to build flexible and re-usable queries. 
```

Comment:
I think maybe it would make more sense without the word "replaceable". I think it's redundant because a parameter is a variable that helps define some kind of function (in this case a query), so the name of the parameter in the query definition is meant to be replaced by some value, which I think is what this feature does. (I.e. you follow the syntax explained below to define how the parameter gets its value, and then the query runs with that value. )

Reply:
```suggestion
Instead of hard-coding alert and event values into Osquery queries, you can use placeholder fields to dynamically pass this data into queries. Placeholder fields function like parameters. You can use placeholder fields to build flexible and re-usable queries. 
```